include/fmt/core.h

```diff
@@ -740,19 +740,27 @@ inline auto get_container(std::back_insert_iterator<Container> it)
 }
 
 template <typename Char, typename InputIt, typename OutputIt>
-FMT_CONSTEXPR auto copy_str(InputIt begin, InputIt end, OutputIt out)
+FMT_CONSTEXPR auto copy_str_constexpr(InputIt begin, InputIt end, OutputIt out)
     -> OutputIt {
   while (begin != end) *out++ = static_cast<Char>(*begin++);
   return out;
 }
 
-template <typename Char, FMT_ENABLE_IF(std::is_same<Char, char>::value)>
-FMT_CONSTEXPR auto copy_str(const Char* begin, const Char* end, Char* out)
-    -> Char* {
+template <typename Char, typename InputIt, typename OutputIt>
+FMT_CONSTEXPR auto copy_str(InputIt begin, InputIt end, OutputIt out)
+    -> OutputIt {
+  return copy_str_constexpr<Char>(begin, end, out);
+}
+
+template <typename Char, typename InputIt, typename OutputIt,
```

Comment:
This is probably nitpicky, but since InputIt and OutputIt aren't iterators in this overload, can you rename the template parameters? I keep getting concerned when I see the "* sizeof(OutputIt)" in the call to memcpy below.

Reply:
Corrected.

Reply:
Thanks!